stdlib/runtime/dataStructures.gr

```diff
@@ -225,3 +225,75 @@ provide primitive tagChar: WasmI32 -> Char = "@tag.char"
  */
 @unsafe
 provide primitive untagChar: Char -> WasmI32 = "@untag.char"
+
+/**
+ * Tag an int8.
+ *
+ * @param num: The int8 to tag
+ * @returns The tagged int8
+ */
+@unsafe
+provide primitive tagInt8: WasmI32 -> Int8 = "@tag.int8"
```

Comment:
We probably want to land this before #1667 so it can be updated for these new primitives. Otherwise you'll need to make the same changes as that PR.

Reply:
I queued that change before I saw this, but once it gets in I can update this PR.

Reply:
@ospencer I can go ahead and update them since I have to rebase anyways